src/ansys_sphinx_theme/static/404.rst

```diff
@@ -1,7 +1,6 @@
-:orphan:
-
 Oops!
 =====
+
 This is unexpected. The page you are requesting does not exist.
 
-If this page should exist, please contact `{{ theme_contact_mail }} <mailto:{{ theme_contact_mail }}`_.
+If this page should exist, please contact `{{ theme_contact_mail }} <{{ theme_contact_mail }}>`_.
```

Comment:
Is there any reason behind removing the `mailto` directive and the `:orphan:` one, @Revathyvenugopal162?

Reply:
Yes,
1. The 404-page html is not rendered as expected. you can check it https://sphinxdocs.ansys.com/404.html. i am expecting, the toc tree, logo and nav bar on the page, so i will open another PR for customized 404 page in which can redirect for edit this page for API/ examples.
2. We are not putting the 404.rst page in our doc, rather we are importing from ansys-sphinx-theme, later rendered to an html, so if we are giving/not giving orphan, doesnt matter, since we are giving the directive orphan to silence the spurious `` document isn't included in any toctree`` warning. and it is directly rendering as 404 html, not 404.rst

Reply:
Thanks a lot for the detailed explanation on the 404 page, @Revathyvenugopal162.

What about the `mailto` directive? Did it introduce any issues?

Reply:
it is also not rendering as expected, not giving the href for the link.